src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SuggestionModeCompletionProviderTests.cs

```diff
@@ -391,7 +391,7 @@ static void Main(string[] args)
         Console.CancelKeyPress += new ConsoleCancelEventHandler(((a$$
     }
 }";
-            await VerifyBuilderAsync(markup);
+            await VerifyNotBuilderAsync(markup);
```

Comment:
don't we still want a builder here?  This could be a parenthesized lambda you're writign.  This seems like something regressed.